docs/schema.md

```diff
@@ -752,9 +754,28 @@ is required for each `credits` entry.
 Each `credits[].contact[]` entry should be a valid, fully qualified, plain-text URL
 at which the credited can be reached. Providing contacts is optional.
 
+### credits[].type[] field
+
+The `credits[].type[]` field should specify type or role of the individual or entity
+being credited.  It must be one of the following defined credit types:
+
+- `FINDER`
+- `REPORTER`
+- `ANALYST`
+- `COORDINATOR`
+- `REMEDIATION DEVELOPER`
+- `REMEDIATION REVIEWER`
+- `REMEDIATION VERIFIER`
+- `TOOL`
+- `SPONSOR`
+- `OTHER`
+
+These values and their definitions correspond directly to the credit types defined in the
+[MITRE CVE specification](https://github.com/CVEProject/cve-schema/blob/master/schema/v5.0/CVE_JSON_5.0_schema.json#L997-L1006).
```

Comment:
Can we link to something a bit more reliable in the case of line number changes? 

The descriptions of the types themselves are also very hard to read from the linked JSON. Is this rendered as a human readable web page somewhere? 

Or alternatively, we can just copy the descriptions here inline, which may be more resilient to changes upstream in CVE.

Reply:
Added the descriptions and updated the link to point to the rendered docs!